python/its-quadkeys/quadkeys-test

```diff
@@ -357,6 +404,20 @@ def test_quadzone():
     check_zone(z2, expected_xor)
     print("OK")
 
+    earth = its_quadkeys.QuadZone("0", "1", "2", "3")
```

Comment:
`    earth = its_quadkeys.QuadZone(["0", "1", "2", "3"])`

to fix the problem:

```
Expected type 'list[QuadKey | str | list[QuadKey | str]]', got 'str' instead
```

Reply:
The signature is supposed to accept postional arguments that are each a string QuadKey, so if something is to be fixed, it is the type hints, not the caller, because that way of calling is explicitly supported (and working, as the regression test-suite demonstrates!).

I was just looking at type hints for `*args` moments ago, and indeed the current type hints are wrong. For `*args` (and `**kwargs`), the type should be about the indivudual parameters, not about `*args` itself: `*args` is a list, but the type hint refers to elements in that list.

So, I think the type hint should be:
```
def __init__(self, *args: QuadKey | str | list[QuadKey | str]):
```

Reply:
BTW, checking type hints is currently not documented (and is not part of the PR tests). How do you actually check the type hints? Would you care to update the workflows to perform such checks?

Reply:
My Pycharm is processing black (Code Formatter) and pep-8 checkes, with a default not defined configuration. Not mandatory but often helpful for the the (optional) pseudotype declation of the signatures (functions, main), for our code...but also the used code :D. We could decide to add to the workflow a (low level, simple, tolerant) pep-8 check, why not ...